scripts/ci/run-mmrelay-meshtasticd-integration.sh

```
@@ -0,0 +1,1484 @@
+
+value = data.get(key)
+if value is None:
+    raise SystemExit(1)
+print(value)
+PY
+}
+
+json_extract() {
+	local json_payload=$1
+	local key_path=$2
+	"${PYTHON_BIN}" - "${key_path}" <<'PY' <<<"${json_payload}"
```

Shellcheck (0.11.0)

[error] Multiple redirections compete for stdin. Use cat, tee, or pass filenames instead.

(SC2261)